Automattic/jetpack
# VideoPress: Load data at `EditDetails` page

Implement selector and load data from backend at `Edit Details` page.

Closes #26152 

#### Changes proposed in this Pull Request:

* Introduce `getVideo` selector and resolver.
* Introduce `useVideo` hook.
* Add `onVideoDetailsClick` handler at `VideoGrid`.
* Redirect to `Edit Details` page from videos items.
* Redirect back to `home` from `Edit Details`.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






_From VideoPress dashboard_

* Start `VideoPress` plugin.
* Go to `VideoPress` dashboard.
* Hover one video
* Click on `Edit details` button.
* You should see the `Edit Details` page with real data.

_From Library_

* Navigate to `Library`.
* Click in one `VideoPress` attachment.
* Click in `Edit details`.
* You should see the `Edit Details` page with real data.

#### Demo

_From VP Dashboard_

https://user-images.githubusercontent.com/1663717/190021818-68f58e2b-024c-4571-801e-628b1a2e4c96.mov

_From Media Library_

https://user-images.githubusercontent.com/1663717/190021857-6c3d8ab3-8130-45c1-9014-fbc422d07ff1.mov
